compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1949,15 +1948,23 @@ private STNode parseParameterRhs(SyntaxKind tokenKind) {
      */
     private STNode parseParameter(SyntaxKind prevParamKind, boolean isParamNameOptional) {
         STNode annots;
+        STNode inclusionSymbol;
         STToken nextToken = peek();
         switch (nextToken.kind) {
             case AT_TOKEN:
+                inclusionSymbol = STNodeFactory.createEmptyNode();
```

Comment:
Will this support `function foo(@someAnnot *Bar b)`?

Reply:
Fixed. Now it works with `annotations` also.